test/test-app/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
let's name x_x_x.ts

app_routing_module.ts

if Module is for a specific component, we don't need an extra module file.